test/test_torch.py

```diff
@@ -2480,6 +2480,21 @@ def test_numel(self):
         self.assertEqual(b.nelement(), 3 * 100 * 100)
         self.assertEqual(b.numel(), 3 * 100 * 100)
 
+    def test_nbytes(self):
+        self.assertEqual(torch.randn(3, dtype=torch.float32).nbytes, 12)
+        self.assertEqual(torch.randn(3, dtype=torch.float64).nbytes, 24)
+
+    def test_itemsize(self):
```

Comment:
can you just move this to test_element_size?  It's fine to test itemsize in test_element_size since they are the same thing.

Reply:
test_element_size was missing a test for float16's so I added that too